src/feedback/components/AmountPicker.vue

```diff
@@ -47,25 +47,33 @@
         <QSlider
           :value="limitedValue"
           :min="0"
-          :max="70"
+          :max="100"
           :step="0.5"
           label
-          class="self-center q-mx-md"
+          class="self-center q-mr-lg"
+          style="margin-left: 8px"
           @input="$emit('input', arguments[0])"
         />
         <!-- don't use type="number" here because browsers might enforce different decimal setting
         depending on browser locale-->
         <QInput
           v-model="valueToNumber"
-          size="4"
-          class="q-mr-md"
+          dense
+          outlined
+          size="8"
         >
           <template #append>
             <span class="text-caption">kg</span>
           </template>
         </QInput>
       </div>
     </div>
+    <div
+      class="content-more-weight-info"
+      :class="{'is-visible': isOnMaxOfSlider}"
+    >
+      <span v-if="isOnMaxOfSlider">{{ $t('ACTIVITY_FEEDBACK.INCREASE_AMOUNT_HINT') }}</span>
```

Comment:
How come we need the `is-visible` _and_ the `v-if`? Can we not `v-if` the whole `div`? (or conversely always show the HINT.. assuming the `is-visible` takes care of showing it or not). Maybe I'm missing something.

Reply:
We removed `is-visible` and used the quasar class `bg-info`. What do you think?

Reply:
(Ah, also now I tried it out, I see the reason for not having the `v-if` around the whole thing - to reserve the space for the info message so it doesn't make the content below it jump when it's visible)

The `bg-info` seems a good solution!